src/clojure/nrepl/bencode.clj

```diff
@@ -389,9 +389,23 @@
 
 (declare lexicographically)
 
+(defn #^{:private true} thing>string
```

Comment:
Isn't it more idiomatic to use naming like `thing->string`? And `defn-`?

Reply:
Ah, I see you just kept the to style in this file. That makes sense. 

Btw, you'll also have to apply this change to the standalone `nrepl/bencode` project.

Reply:
I'll update the standalone project once done.